### PR TITLE
Delete ooni_home_autorun on hard reset

### PR DESCRIPTION
**Pull request by majakomel**
Closes https://github.com/ooni/probe/issues/2071

Do we also need to stop autorun at this point?

### CODE REVIEW

File: main/actions.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 const { Ooniprobe } = require('./utils/ooni/ooniprobe')
+const { getAutorunHomeDir } = require('./utils/paths')
+const fs = require('fs-extra')
 const log = require('electron-log')
 const Sentry = require('@sentry/electron')
 
 const hardReset = () => {
   const ooni = new Ooniprobe()
   log.info('hardReset: performing a hard reset of the installation')
+  fs.removeSync(getAutorunHomeDir())
```
Comment (hellais) [inline main/actions.js:10]: I think that what we ought to be doing is calling `ooniprobe reset --force` with the `OONI_HOME` environment variable set to `getAutorunHomeDir`.

That way, if there is any additional cleanup operation that needs to happen on hard reset, we can implement it inside of probe-cli directly and not have to remember to keep it in sync between the two directories.

**Comment by hellais** [comment]
> Do we also need to stop autorun at this point?

I don't think we should do that. The autorun home directory will be re-created automatically the next time it runs, but it will be empty.

I guess one thing we need to think about is what is going to happen if the hard reset is executed while the autorun task is currently running.

**Comment by bassosimone** [comment]
> I guess one thing we need to think about is what is going to happen if the hard reset is executed while the autorun task is currently running.

Yeah, this would be an interesting scenario

**Review by bassosimone — COMMENTED** [review]
The diff looks good to me. I have spent some time thinking about what could happen if a hard reset occurs while an autorun is running, and I don't know what could happen. 🤷

**Pull request merged**